augur/application/cli/collection.py

```diff
@@ -22,14 +22,15 @@
 from augur.application.db.session import DatabaseSession
 from augur.application.logs import AugurLogger
 from augur.application.db.lib import get_value
-from augur.application.cli import test_connection, test_db_connection, with_database
+from augur.application.cli import test_connection, test_db_connection, with_database, DatabaseContext
```

Comment:
Is this one just incorporating the latest from @ABrain7710 ?

Reply:
No, the changes are unrelated